docs/04-Bar-Chart.md

```diff
@@ -49,6 +49,7 @@ borderSkipped | `String or Array<String>` | Which edge to skip drawing the borde
 hoverBackgroundColor | `Color or Array<Color>` | Bar background color when hovered
 hoverBorderColor | `Color or Array<Color>` | Bar border color when hovered
 hoverBorderWidth | `Number or Array<Number>` | Border width of bar when hovered
+stack | `String` | For grouping stacked bars
```

Comment:
> I was thinking group first, but finally stack might be better. 

After I chose `stack`, I was worried that it would be very bar specific (just in case we want to use the same property for grouping datasets for other chart types) + we are calling it **group**ed stacked bars. But I'll go with whatever you suggest (I've being going back and forth between them that they both seem logical - I'll have good things to say about `stack` once we've changed to `group` :-))

> Can we explicitly document this dataset.stack as an ID and not a name to avoid any confusion with label/text/title?

You mean call it stackID (like we do with axisIDs) and point to options (in future) for name, styling? Or do you mean just change `For grouping stacked bars` to `ID for grouping stacked bars`? Should be simple enough either way.